pkg/cmd/dev/build.go

```diff
@@ -34,6 +34,10 @@ type buildTarget struct {
 	isGoBinary bool
 }
 
+// devVersion (YYYYMMDD) is used to stamp dev-built dev binaries. It has to be
+// kept in sync with bin/dev; look there for more details.
+const devVersion = 20220117
```

Comment:
I really dislike having to keep this in sync in two places. I think that realistically this number will have to be bumped quite frequently, so making it difficult to mess up would be good.

You can check out what we do in `builder.go`, which is open `build/teamcity-bazel-support.sh` to read out the Docker image.  You can do something similar here.

Reply:
Done.